scripts/lint-dotnet-worker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+#
+# Script to lint .NET code
+#
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(dirname "$SCRIPT_DIR")"
+
+echo "Linting .NET..."
+
+if ! command -v dotnet &> /dev/null; then
+    echo "Error: .NET is not installed. Please install .NET first."
+    exit 1
+fi
+
+echo ".NET version: $(dotnet --version)"
+
+cd "$REPO_ROOT/workers/dotnet"
+
+echo "Cleaning .NET workspace..."
+dotnet clean
```

Comment:
Not sure why all these clean first, seems unnecessary.

Reply:
I had an issue that I thought was related to a partial build; but I'm happy to remove it again since that might have just been a fluke. I'd expect the respective tools to behave reasonably well with partial builds.